core/src/main/java/org/apache/struts2/StrutsConstants.java

```diff
@@ -250,7 +250,7 @@
     public static final String STRUTS_EL_THROW_EXCEPTION = "struts.el.throwExceptionOnFailure";
 
     /** Logs properties that are not found (very verbose) */
-    public static final String STRUTS_LOG_MISSING_PROPERTIES = "struts.ognl.logMissingProperties";
```

Comment:
Hi Lukasz.
It is possible (if unlikely) that someone has been referencing `STRUTS_LOG_MISSING_PROPERTIES` from the constants file, and removing it will cause breakage.
Probably not a big concern, but maybe consider leave the old constant, have it set to the new one's value, and just mark the old one as deprecated ?
Something like:
```
/** Logs properties that are not found (very verbose) */
@deprecated as of 2.6.  Use {@link STRUTS_OGNL_LOG_MISSING_PROPERTIES} instead.
public static final String STRUTS_LOG_MISSING_PROPERTIES = STRUTS_OGNL_LOG_MISSING_PROPERTIES;

/** Logs properties that are not found (very verbose) */
public static final String STRUTS_OGNL_LOG_MISSING_PROPERTIES = "struts.ognl.logMissingProperties";
```
It might not really be necessary, but just something to consider.  :)

Reply:
Done